frontend/src/styles/login-page.scss

```diff
@@ -26,3 +26,8 @@
         }
     }
 }
+
+#login-error {
```

Comment:
Spaces. :) Plus, let's work on the styling of this.

  * There is a red color defined in `colors.scss`, let's use that. (That file came from me tweaking some online color scheme generator for a somewhat pleasant effect, feel free to modify but not the most important issue). 
  * The error message should have some kind of background. 
  * I think it'd be good to position it above content, just below the menu so it'd be suitable to display any error we have in the app. 
  * Also we could try to slide it in, see how it looks. 
  * Let's experiment with font-sizes as well (I see that you did that, and I like it, it just might look different when you move the whole error message around)

Making an error component will mean that there this will go to an `error.scss` file or something similar. What you did is good here, let's just push it to one step futher where it's awesome for the longer-term vision. :)

Reply:
Thanks for the detailed comments! What do you mean 'above content and below the menu'? Do you mean below the login and register buttons?

Reply:
Well I meant after logging in, it could be under the top menu bar, the one described in `Menu.js`. That is present everywhere except the very beginning in the main screen, where at this point there is not much which could go wrong (aside from fatal network fault or something). So we can cross that particular bridge when we get there, but if you'd make an error component conditionally appearing under the `Menu` component, that'd take us 95% of the way